validator/src/main.rs

```diff
@@ -266,7 +263,7 @@ pub fn main() {
                 .long("vote-account")
                 .value_name("PUBKEY")
                 .takes_value(true)
-                .validator(is_keypair)
+                .validator(is_pubkey_or_keypair)
```

Comment:
Contrary to [this help message](https://github.com/solana-labs/solana/pull/6790/files#diff-1f422d7e73618aca7bce9935356de012R264), `--vote-account` never accepted a pubkey it seems...

Reply:
This is fixed by https://github.com/solana-labs/solana/pull/6812